pages/common/hg-update.md

```diff
@@ -0,0 +1,19 @@
+# hg update
+
+> Update the working directory to a specified changeset.
+
+- Update to the tip of the current branch:
+
+`hg update`
+
+- Update to the specified revision:
+
+`hg update --rev {{revision}}`
+
+- Update and discard uncommitted changes:
+
+`hg update --clean`
+
+- Update to the last commit matching a specified date:
+
+`hg update --date {{date}}`
```

Comment:
Do you think an example date will be better ? Often times I am not sure what date format to give as different tools accept different format.

Reply:
I guess so. Would something like `dd-mm-yyyy` do?

Reply:
Sure.

Reply:
Just to check, should it be added in the description or in the example?

Like, `hg update --date {{dd-mm-yyyy}}`

Reply:
example. I have updated it myself.

Reply:
Cheers 👍